src/scripts/main.js

```diff
@@ -1 +1,35 @@
 'use strict';
+
+const links = document.querySelectorAll('.list-item__link');
+// const images = document.querySelectorAll('.gallery__thumb');
+const largeImg = document.querySelector('#largeImg');
+
+for (let i = 0; i < links.length; i++) {
+  links[i].removeAttribute('href');
+
+  let number = '';
+
+  switch (i) {
+    case 0:
+      number = 'first';
+      break;
+    case 1:
+      number = 'second';
+      break;
+    case 2:
+      number = 'third';
+      break;
+    case 3:
+      number = 'fourth';
+      break;
+    case 4:
+      number = 'fifth';
+      break;
+  }
+
+  const source = `http://localhost:8080/images/landscape-${number}.png`;
+
+  links[i].addEventListener('click', () => {
```

Comment:
Instead of adding event handler to each link separatelly, you can add event handler to the link list ([ Check event delegation ](https://www.freecodecamp.org/news/event-delegation-javascript/)) like:
```js
const list = document.getElementById('thumbs');

list.addEventListener('click', (event) => {
const imageLink = event.target.closest('.list-item__link')

// here you can set imageLink.href value to the main gallery image
})
```